index.html

```diff
@@ -144,6 +144,37 @@ <h5>LET'S CONNECT</h5>
           <hr>
         </div>
       </div>
+      <div class="card2">
+        <div class="content4">
+          <h2 class="contact-header">Contact me</h2>
+          <p class="p-contact">
+            If you have an application you are interested in developing, a 
+            feature that you need built or a project that needs coding. I’d 
+            love to help with it
+          </p>
+          <form action="">
+            <ul class="form-ul">
+              <li>
+                <label for="name"></label>
+                <input type="text" id="name"  maxlength="30" name="user_name" size="43" placeholder="Lugard"/><br><br>
+              </li>
+              <li>
+                <label for="mail"></label>
+                <input type="email" id="mail" size="43" name="user_email" placeholder="agulugard@gmail.com"/><br><br>
+              </li>
+              <li>
+                <label for="msg"></label>
+                <textarea name="user_message" id="msg"  maxlength="500" cols="30" rows="10" size="30" placeholder="Write your message here"></textarea>
+              </li>
+              <div class="button">
+              <li>
+                <button type="submit" id="b-contact">Get it Touch</button>
+              </li>
+              </div>
+            </ul>
+          </form>
```

Comment:
- [ ] Kindly ensure that you make use of form spree to act as your form data handler. Kindly create an account at [formspree](https://formspree.io/html/) and use it to add your unique id URL inside the action.
- [ ] I have also realized that I can be able to submit the form contents even when it is empty. This is because thee are no form validations to ensure that I am not able to submit an empty form. Kindly add the required attribute.